tests/test_targets_realistic_coverage.py

```
@@ -123,6 +123,18 @@ def test_bmr_calculations_realistic(self):
                     )
                     pass
 
+                # Test formula selection without body_fat (should return "mifflin")
+                try:
+                    user_data_no_fat = user_data.copy()
+                    user_data_no_fat.pop("body_fat", None)  # Remove body_fat
+                    best_formula_no_fat = get_bmr_formula(user_data_no_fat)
+                    assert best_formula_no_fat == "mifflin"
+                except Exception as e:
```

Ruff (0.13.3)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)